codetransformer/core.py

```diff
@@ -135,7 +136,7 @@ def visit(self, co, *, name=None):
         # must backreference their original jump target before any transforms.
         # Don't refactor this into a single pass.
         self._instrs = tuple(_sparse_args([
-            Instruction(b.opcode, b.arg) for b in Bytecode(co)
+            Instruction.from_opcode(b.opcode)(b.arg) for b in Bytecode(co)
```

Comment:
I'd expect from_opcode to be a smart constructor, not a smart subclass-creator.  Is there a reason this can't be `from_opcode(b.opcode, b.arg)`?